### PR TITLE
Partners Landing Pages

### PR DESCRIPTION
**Pull request by ccorda**

### CODE REVIEW

File: website-guts/templates/components/modals/become_partner_technology.hbs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: modal_wrapper
+wrapper_id: contact-sales
+data_modal_id: contact-sales
+data_modal_name: contact-sales
+form_id: contact-sales-form
+TR_modal_title: Contact Sales
+TR_modal_sub_header: "Please tell us about yourself and your company (all fields required):"
+TR_primary_button_text: Contact Sales
+TR_negative_button_text: Cancel
```
Comment (dtothefp) [inline website-guts/templates/components/modals/become_partner_technology.hbs]: @yuvilio here's your problem, you didn't change the data attributes and modal id. they are all `contact-sales`

they need a unique id to to target with `oForm` and then an unique data attribute to target with our modal js

**Comment by yuvilio** [comment]
retest this please

**Comment by stewsmith** [comment]
@julieritchie Is the copy final on this? If so, we can start translations.

**Comment by stewsmith** [comment]
@KarenLoughrey This is ready for translation. Thanks!

**Comment by KarenLoughrey** [comment]
@stewsmith Translations in progress! Waiting to hear back on deadline confirmation but am hoping they should be ready on Thursday. Will keep you posted!

**Comment by KarenLoughrey** [comment]
@stewsmith Translations are done!

**Pull request merged**